Diijkstra_shortest_algo_in_python.py

```diff
@@ -0,0 +1,97 @@
+from collections import deque, namedtuple
+
+
+# we'll use infinity as a default distance to nodes.
+inf = float('inf')
+Edge = namedtuple('Edge', 'start, end, cost')
+
+
+def make_edge(start, end, cost=1):
+  return Edge(start, end, cost)

```

Comment:
E111 indentation is not a multiple of four

*Origin: PycodestyleBear (E111), Section: `all.autopep8`.*